packaging/ubuntu-20.04/rules

```diff
@@ -193,7 +193,7 @@ override_dh_auto_build:
 	PATH="$$(pwd)/packaging/build-tools/:$$PATH" \
 		dh_auto_build -- -mod=vendor $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS) $(DH_GOPKG)/cmd/...
 
-	(cd _build/bin && GOPATH=$$(pwd)/.. go build -mod=vendor $(BUILDFLAGS) $(GCCGOFLAGS) $(SNAP_TAGS) $(DH_GOPKG)/cmd/snap)
+	(cd _build/bin && GOPATH=$$(pwd)/.. CGO_LDFLAGS="-L$(CURDIR)/usr/lib/$(DEB_HOST_MULTIARCH) -L$(CURDIR)/lib/$(DEB_HOST_MULTIARCH)" go build -mod=vendor $(BUILDFLAGS) $(GCCGOFLAGS) $(SNAP_TAGS) $(DH_GOPKG)/cmd/snap)
```

Comment:
why do we need this change? it seems cross building related?

Reply:
I moved this back into snapcraft.yaml (upon thinking about it this is a bad place) and then added a comment, the naming is confusing sadly, it sounds like it's about cross build but it's not :/